src/codewhisperer/client/codewhisperer.ts

```diff
@@ -197,7 +198,20 @@ export class DefaultCodeWhispererClient {
         const client = await this.createUserSdkClient()
         const requester = async (request: CodeWhispererUserClient.ListAvailableCustomizationsRequest) =>
             client.listAvailableCustomizations(request).promise()
-        return pageableToCollection(requester, {}, 'nextToken').promise()
+        return pageableToCollection(requester, {}, 'nextToken')
+            .promise()
+            .then(resps => {
+                let logStr = 'codewhisperer: listAvailableCustomizations API request:'
+                resps.forEach(resp => {
+                    const requestId = resp.$response.requestId
+                    logStr += `\n${indent('RequestID: ')}${requestId},\n${indent('Customizations:')}`
+                    resp.customizations.forEach((c, index) => {
+                        logStr += `\n       ${index.toString().padStart(2, '0')}: ${c.name?.trim()}`
```

Comment:
could use indent() instead of doing this manually

Reply:
ack, updated